### PR TITLE
The Next Web clone 

### PR DESCRIPTION
**Pull request by vmwhoami**
The goal of this project is to build a responsive clone of [The Next Web website](https://thenextweb.com/)

> In this project, we made use of Grids, Flexbox and Positions and layout elements on the page correctly.

> Media queries were used to target different screen sizes in order for us to effectively and efficiently let the website fit correctly to each screen sizes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,265 +1,733 @@
...
-      href="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/icons/favicon-32x32.png?v=1589790655"
-      type="image/x-icon"
-    />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;300;400;700&display=swap"
-      rel="stylesheet"
-    />
-    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
-    <link rel="stylesheet" href="/css/styles.css" />
```
Comment (trekab) [inline index.html]: Remove the starting `/` from this CSS linking tag. It makes your styles unavailable to the page.

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes Required!
Hi @vmwhoami and @acushlakoncept, well done! :+1: The project looks good! However, there are some core requirements you need to implement to complete this project. These are;
- [x] Stickler isn't enabled for this repo.
- [x] Live Link in the README doesn't lead to the clone page.
- [x] TWN logo should appear at the leftmost part of the navbar on medium screens.
- [x] In the Top 3 articles section, the TWN logo should disappear on medium screens.
- [x] There should be no small breakpoint as it is mobile-first and the base styles are for the mobile screen. ([Screenshot on mobile-first approach](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))
- [x] medium: min-width around 768px
- [x] large: min-width 1024px

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review_ :wink: 
__Treasure Kabareebe__

File: index.html (hunk 1 above)
Comment (acushlakoncept) [reply index.html]: Thank you @trekab for the feedback.
We have implemented the recommended issues and updates.

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes Required!
Hey @vmwhoami and @acushlakoncept, great job implementing the earlier requested changes! :clap: Unfortunately, the new changes you implemented have caused some more minor issues I'd like you to look at.
### Mandatory changes:
- [x] A hamburger icon should appear at the rightmost part of the navbar on small screens(After the search icon). Please compare with [original](https://thenextweb.com/) site.
- [x] Your clones TNW logo looks different from the logo on the [original](https://thenextweb.com/) site on small screens.
- [x] The `Latest news` section and the `Latest deals` section are broken for medium screens.


__N.B:__ When viewing your page through the mobile feature of developer tools, choose the responsive option, and observe your page as you resize it accordingly.

__Kindly work on the above changes and submit another code review request.__
_Feel free to reach out on slack for any inquiries about this review_ :wink:
__Treasure Kabareebe__

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,848 @@
...
+  margin-bottom: 1rem;
+}
+
+.news-item {
+  position: relative;
+}
+
+.news-mobile .news-item {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
```
Comment (rammazzoti2000) [inline css/styles.css]: on medium screens, in `Latest News` and `Latest Deals` your having some space gap alignment issues between the picture and title description
`grid-template-columns: 0.5fr 3fr;` --> should fix it

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,848 @@
...
+
+.top-footer {
+  width: 100%;
+  padding: 4.2rem 0 2.8rem;
+  background-color: var(--grey-2);
+}
+
+.top-footer__social {
+  display: flex;
+  flex-flow: row;
```
Comment (rammazzoti2000) [inline css/styles.css]: It seems the social links in the footer are not aligning on a new row when on small screens
`flex-flow: row wrap;` --> should fix it

**Review by rammazzoti2000 — APPROVED** [review]
## Project Approved :tada: :rocket: 
Hello team great work on the project and also on implementing the changes required be the previous tse
This project is meeting all requirements and is ready for approval, feel free to merge when you are ready
### :exclamation: Before Merging [required]: 
- [x] In the `Latest News` & `Latest deals`, on medium screens. you have some space gap alignment issues between the picture and the title description, you can easily fix that by reducing the fraction size on the first column to 0.5fr, see the inline comment
image
- [x] The social links in the footer should use `flex-flow: row wrap;` so that they can wrap on a new line when resizing on small screens
image
###
Cheers and Happy coding! :clap: 

##### Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>
### :heavy_check_mark: Status: Approved

File: css/styles.css (hunk 2 above)
Comment (acushlakoncept) [reply css/styles.css]: Thank you @rammazzoti2000 for your keen eye 👍 on details.
The issue was fixed using the your recommendations ⭐.

File: css/styles.css (hunk 3 above)
Comment (acushlakoncept) [reply css/styles.css]: Yes it did 👍 
Thank you. 💯 🙏

**Pull request merged**